### DISCUSSION TITLE
通过系统镜像升级系统版本导致pcl个性化中隐藏主题丢失

### DISCUSSION THREAD

**Discussion by cucumbertw** [Meloong-Git/PCL#4647] [讨论]
用iso升级了一下系统，从win10卡到1890的更新升级到22h2，导致识别码变动，pcl内的隐藏主题都没了 555~
之前已经全解锁了

**Comment by cucumbertw** [comment]
想问一下还有救吗

**Reply by tangge233** [reply]
有备份么？没有没救

**Comment by WForst-Breeze** [comment]
隐藏主题没了是没有办法还原的，只能手动重新解锁，活跃橙可以在你提交的 **符合条件** Issue 下方再次发送识别码解锁 🧐
